ext/mbstring/libmbfl/filters/mbfilter_cp5022x.c

```diff
@@ -40,6 +40,9 @@ static void mb_wchar_to_cp50220(uint32_t *in, size_t len, mb_convert_buf *buf, b
 static void mb_wchar_to_cp50221(uint32_t *in, size_t len, mb_convert_buf *buf, bool end);
 static void mb_wchar_to_cp50222(uint32_t *in, size_t len, mb_convert_buf *buf, bool end);
 
+/* See mbstring.c */
+uint32_t mb_convert_kana_codepoint(uint32_t c, uint32_t next, bool *consumed, uint32_t *second, int mode);
```

Comment:
This is introducing a reverse-dependency from libmbfl to mbstring. Probably the function should continue to live in libmbfl?

Reply:
Oh, indeed. @alexdowad, can you please address this? If `mb_convert_kana_codepoint()` can be easily changed to be moved to libmbfl, an option might be to establish a callback.

Reply:
Will do (in my next PR, which is close to ready).